### PR TITLE
 math-magicians: setup

### PR DESCRIPTION
**Pull request by Mhdez221993**
General requirements
- Make sure that there are no linter errors.
- Make sure that you used the correct Gitflow.
- Make sure that you documented your work in a professional way.

JavaScript requirements
- Follow our list of best practices for JavaScript.
 
Project requirements
- Create a React application is using Create React App (CRA) and following the instructions in this guide.
- The application should run in the browser without errors.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById('root'),
```
Comment (vichuge) [inline src/index.js:11]: - [ ] I download the repo, then I go to `main` branch, delete the `package-lock.json` file and then I use `npm i` to install the packages, after that I use `npm start` to check the deploy and this error appears, please work on that. 🐶 
image

File: package.json [hunk 2]
```diff
@@ -40,6 +40,18 @@
     ]
   },
   "devDependencies": {
-    "cypress": "^8.2.0"
+    "babel-eslint": "^10.1.0",
+    "cypress": "^8.2.0",
```
Comment (vichuge) [inline package.json:44]: - [OPTIONAL] The requirements for this project don't say anything about doesn't use another tools but, is really necessary to use cypress? Please, consider removing this. 🐱

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Mhdez221993) [reply src/index.js:11]: Thank you for the comment, it'll be fix soon.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Mhdez221993  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**